frontend/src/main/pages/UCSBDates/UCSBDatesEditPage.js

```diff
@@ -50,15 +57,12 @@ export default function UCSBDatesEditPage() {
   }
 
   return (
-    <>
-      { (status === 'success') && ucsbDate && (
```

Comment:
I'm a bit unfamiliar with this code so just checking, did you intend to remove this check?

Otherwise looks good